packages/mobile/src/screens/sign-on-screen/components/AudiusValues.tsx

```diff
@@ -1,26 +1,24 @@
-import type { SvgProps } from 'react-native-svg'
+import Animated, { FadeIn, FadeOut } from 'react-native-reanimated'
 
-import type { FlexProps } from '@audius/harmony-native'
+import type { Icon } from '@audius/harmony-native'
 import { Flex, Text } from '@audius/harmony-native'
 
+import { PANEL_EXPAND_DELAY } from '../constants'
+
 import IconCloudUpload from './temp-harmony/iconCloudArrowUp.svg'
 import IconHeadphones from './temp-harmony/iconHeadphones.svg'
 import IconMessage from './temp-harmony/iconMessages.svg'
```

Comment:
These are safe to import from harmony-native now I assume if you want to add that in

Reply:
oh yess! 100%